packages/site/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.4.0]
+
+### Added
+
+- Added localhost to ChainConfig ([#120](https://github.com/MetaMask/snap-account-abstraction-keyring/pull/120))
+
```

Comment:
Not sure I got the whole context of that PR, but maybe something like this might be better, WDYT?

(I might have missed the order, so you will have to run `prettier` probably)

```suggestion
### Changed

- Better user feedback if selected chain ID is not correct ([#120](https://github.com/MetaMask/snap-account-abstraction-keyring/pull/120))
  - It nows display a red border in case of error

### Fixed

- Fixed issues with test IDs, context usage and state ([#120](https://github.com/MetaMask/snap-account-abstraction-keyring/pull/120))

```